src/Storages/LiveView/StorageLiveView.cpp

```diff
@@ -278,6 +278,40 @@ StorageLiveView::StorageLiveView(
     active_ptr = std::make_shared<bool>(true);
 }
 
+BlockOutputStreamPtr StorageLiveView::tryGetTargetTableOutputStream(const Context & context) const
+{
+    auto storage = tryGetTargetTable();
+    BlockOutputStreamPtr out = nullptr;
+
+    if (storage)
+    {
```

Comment:
I think this code is a copy from `InterpreterInsertQuery`. Can't we use it here directly?
If not, maybe we can extract common logic to separate place?

Reply:
I will check if we can use InterpreterInsertQuery.